nautobot/core/templates/components/panel/footer_table.html

```diff
@@ -0,0 +1,17 @@
+{% block footer_left %}
+    {% if body_content_table_list_url and more_queryset_count > 0 %}
+        <a href="{{ body_content_table_list_url }}" class="btn btn-default btn-xs">
+            <span class="mdi mdi-database-search-outline" aria-hidden="true"></span> View {{ more_queryset_count }} more object(s)
```

Comment:
Maybe `mdi-dots-vertical` or `mdi-dots-horizontal` instead? Or no icon at all? This isn't a search operation per se so I find `mdi-database-search-outline` a bit unintuitive.

Reply:
Also wondering if it really should be a button, or whether it should just be a normal hyperlink?

Reply:
Also, can we have this use `body_content_table_verbose_name`/`body_content_table_verbose_name_plural` in place of the generic `object(s)`?